Core/clim-core/frames.lisp

```diff
@@ -830,96 +830,131 @@ documentation produced by presentations.")
        ;; Update frame-current-panes and the special pane slots.
        (update-frame-pane-lists frame))))
 
+(defun parse-define-application-frame-options (options)
+  (let ((infos '(;; CLIM
+                 (:pane                  * :conflicts (:panes :layouts))
+                 (:panes                 * :conflicts (:pane))
+                 (:layouts               * :conflicts (:pane))
+                 (:command-table         1)
+                 (:command-definer       1)
+                 (:menu-bar              ensure-1)
+                 (:disabled-commands     *)
+                 (:top-level             1)
+                 ;; icon
+                 (:geometry              *)
+                 ;; resize-frame
```

Comment:
ditto

Reply:
An annotation for the spec entry for `define-application-frame` states that this should probably be an accepted option, but we don't seem to support it.

Reply:
I added a comment explaining this to the commit.